test/HelixTasks/SDKCustomCreateXUnitWorkItemsWithTestExclusion.cs

```diff
@@ -22,7 +22,7 @@ public class SDKCustomCreateXUnitWorkItemsWithTestExclusion : Build.Utilities.Ta
         /// The two required parameters will be automatically created if XUnitProject.Identity is set to the path of the XUnit csproj file
         /// </summary>
         [Required]
-        public ITaskItem[] XUnitProjects { get; set; }
+        public ITaskItem[]? XUnitProjects { get; set; }
```

Comment:
@rainersigwald Have ya'll enabled nullable for any msbuild tasks? Is it possible for these to be null or does MSBuild force them to have a value always? Do they have to be marked Required to not be null?